src/fu-util-common.h

```diff
@@ -94,3 +94,5 @@ gboolean	 fu_util_send_report		(FwupdClient	*client,
 						 GError		**error);
 gint		 fu_util_sort_devices_by_flags_cb	(gconstpointer	 a,
 						 gconstpointer	 b);
+gint		 fu_util_activate_task_sort_cb	(gconstpointer a,
```

Comment:
is there anything activation or task-specific about this? `fu_util_activate_task_sort_cb` should probably be something like `fu_util_device_order_sort_cb` and `fu_util_activate_task_compare` maybe something like `fu_util_device_order_compare`

Reply:
no, nothing activation specific.  will rename them both.

Reply:
fixed